src/en/community/events/2023/ceph-days-korea/index.md

```diff
@@ -21,9 +24,6 @@ join forces to discuss things like the status of the Ceph project, recent Ceph
 project improvements and roadmap, and Ceph community news. The day ends with
 a networking reception, to foster more Ceph learning.
 
-<a class="button" href="#">CFP Coming Soon</a>
-<a class="button" href="#">Registeration Coming Soon</a>
-
 ## Important Dates
 
 - CFP Opens: Coming Soon!
```

Comment:
This should be 2023-02-24 with this change?

Reply:
Updated, thank you.